src/it/osgi/pom.xml

```diff
@@ -13,6 +13,18 @@
   </properties>
   
   <profiles>
```

Comment:
There's a few missing or incorrect ones, use these ones for consistency at least:
https://github.com/bytedeco/javacpp-presets/blob/master/pom.xml#L1914
The name of the DLL on Windows is also incorrect.